pkg/utils/cluster_info.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/klog"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	clusterIDConfMap = "cluster-id"
+	// Todo: Is there a place where i can get this?
+	configMapNamespace = "liqo"
```

Comment:
This should be resolved before merging

Reply:
Yes, I have already talked with Alex about this, ATM it's ok to have a costant for it.